package.json

```diff
@@ -26,7 +26,7 @@
     "express": "^4.15.3",
     "github": "^9.2.0",
     "github-app": "^3.0.0",
-    "github-webhook-handler": "^0.6.0",
+    "github-webhook-handler": "rvagg/github-webhook-handler#v0.6.1",
```

Comment:
I'm not thrilled about depending on an unreleased package. It's probably not a huge issue, but it gives me pause. Here are a few alternatives we could consider:

1. Push a fork to a [scoped package](https://docs.npmjs.com/getting-started/scoped-packages) (e.g. `@probot/github-webhook-handler`

2. Now that the web server is using express, we could use it to limit the webhook to post requests (I think we can replace `app.use(webhook)` with `app.post(path, webhook)` and it will Just Work™)

What do yo think? Are either of those worth it?